.github/workflows/build-test-distribute.yaml

```diff
@@ -75,6 +78,9 @@ jobs:
           echo "images=$(make images/info/release/json)" >> $GITHUB_OUTPUT
           echo "registry=$(make docker/info/registry)" >> $GITHUB_OUTPUT
           echo "version=$(make build/info/version)" >> $GITHUB_OUTPUT
+          echo "distribution_repository=$(make build/info/distribution/repo)" >> $GITHUB_OUTPUT
+          echo "security_assets_package_name=$(make build/info/distribution/security/pkg)" >> $GITHUB_OUTPUT
```

Comment:
No need to make things complex for no reason. Make this static and just in the `distributions` step

Reply:
done